doc/project/code_of_conduct.rst

```diff
@@ -1,148 +1,52 @@
+website https://numfocus.org/code-of-conduct
 
-**Community Impact**: Use of inappropriate language or other behavior deemed
-unprofessional or unwelcome in the community.
-
-**Consequence**: A private, written warning from community leaders, providing
-clarity around the nature of the violation and an explanation of why the
-behavior was inappropriate. A public apology may be requested.
-
-2. Warning
-----------
-
-**Community Impact**: A violation through a single incident or series
-of actions.
-
-**Consequence**: A warning with consequences for continued behavior. No
-interaction with the people involved, including unsolicited interaction with
-those enforcing the Code of Conduct, for a specified period of time. This
-includes avoiding interactions in community spaces as well as external channels
-like social media. Violating these terms may lead to a temporary or
-permanent ban.
-
-3. Temporary Ban
-----------------
-
-**Community Impact**: A serious violation of community standards, including
-sustained inappropriate behavior.
-
-**Consequence**: A temporary ban from any sort of interaction or public
-communication with the community for a specified period of time. No public or
-private interaction with the people involved, including unsolicited interaction
-with those enforcing the Code of Conduct, is allowed during this period.
-Violating these terms may lead to a permanent ban.
-
-4. Permanent Ban
-----------------
-
-**Community Impact**: Demonstrating a pattern of violation of community
-standards, including sustained inappropriate behavior,  harassment of an
-individual, or aggression toward or disparagement of classes of individuals.
-
-**Consequence**: A permanent ban from any sort of public interaction within
-the community.
-
-Attribution
-===========
+HOW TO REPORT
+=============
 
-This Code of Conduct is adapted from the `Contributor Covenant <https://www.contributor-covenant.org>`_,
-version 2.0, available at
-https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+If you feel that the Code of Conduct has been violated, feel free to
+submit a report, by using the form: `NumFOCUS Code of Conduct Reporting
+Form <https://numfocus.typeform.com/to/ynjGdT?typeform-source=numfocus.org>`__
 
-Community Impact Guidelines were inspired by `Mozilla's code of conduct
-enforcement ladder <https://github.com/mozilla/diversity>`_.
+WHO WILL RECEIVE YOUR REPORT
+============================
 
+Your report will be received and handled by NumFOCUS Code of Conduct
+Working Group; trained, and experienced contributors with diverse
+backgrounds. The group is making decisions independently from the
+project, PyData, NumFOCUS or any other organization.
 
-For answers to common questions about this code of conduct, see the FAQ at
-https://www.contributor-covenant.org/faq. Translations are available at
-https://www.contributor-covenant.org/translations.
+You can learn more about the current group members, as well as the
```

Comment:
-> "For a comprehensive description of the NumFOCUS Code of Conduct reporting and report handling procedures, including the current membership of the Working Group, refer to the `NumFOCUS Code of Conduct web page <https://numfocus.org/code-of-conduct>`__."